synapseclient/table.py

```diff
@@ -829,75 +857,155 @@ class Dataset(SchemaBase):
     :py:classmethod:`synapseclient.client.create_snapshot_version`.
 
     Example::
+
         syn = synapseclient.login()
         syn.create_snapshot_version(
             dataset.id,
             label="v1.0",
             comment="This is version 1")
     """
     _synapse_entity_type: str = "org.sagebionetworks.repo.model.table.Dataset"
-    _property_keys: List[str] = SchemaBase._property_keys + ['dataset_items']
+    _property_keys: List[str] = SchemaBase._property_keys + ['datasetItems']
```

Comment:
Unfortunately, I used `datasetItems` in the previous PR but had forgotten, so I had to address it here as well.  Otherwise, the methods would not work as expected.

Reply:
[Done](https://github.com/Sage-Bionetworks/synapsePythonClient/pull/918).